ivy_tests/test_ivy/test_frontends/test_paddle/test_paddle_fft.py

```diff
@@ -1,3 +1,38 @@
 # global
+from hypothesis import strategies as st
 
 # local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+@handle_frontend_test(
+    fn_tree="paddle.fft.fft",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("valid"),
+        shape=(2,),
```

Comment:
Please widen the shape range (their example is `(7,)`) :1234:

Reply:
I've widened the range

Reply:
Does this work for any `int` from 2 to 7? Maybe we should do 
`shape=st.sampled_from(None | st.integers)`

Reply:
I've removed `shape` and added `None` for `n`